### PR TITLE
Redirect security reports to github mechanism (rather than discourse)

### PR DESCRIPTION
**Pull request by denis-yuen**
Redirecting policy as a part of https://ucsc-cgl.atlassian.net/browse/SEAB-4291

### CODE REVIEW

**Comment by denis-yuen** [comment]
Note, I already edited https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506

**Comment by denis-yuen** [comment]
See also https://github.com/dockstore/dockstore-documentation/pull/267

File: SECURITY.md [hunk 1]
```diff
@@ -7,15 +7,4 @@ released tagged minor version. All previous versions are unsupported.
...
-
-1. Navigate to [Discourse](https://discuss.dockstore.org/) and login.
-2. Select your profile icon, located in the top right corner of the screen.
-3. Select the `mail` icon, located in the dropdown.
-4. Send a message to the `dockstore_admins` group.
-
-Note
-
-> If you are unable to see a New Message button on the mail page, you may be considered a new user and have insufficient privileges. Entering 5 topics and viewing 30 posts over a minimum of 10 minutes will raise your privileges. You will be notified of any privilege changes to your account via the mailbox.
+Use the "Report a vulnerability" button to the upper right to report security vulnerabilities. A helpdesk ticket on [discourse](https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506) may be an option for privacy complaints or other urgent non-security matters. Both options will be addressed by Dockstore administrators.
```
Comment (coverbeck) [inline SECURITY.md]: > A helpdesk ticket on [discourse]...

I suggest not saying `discourse`... If you're not familiar with discourse (I wasn't before I joined the Dockstore team), and you go to our site, it doesn't say `discourse`.

**Review by coverbeck — APPROVED** [review]

File: SECURITY.md [hunk 2]
```diff
@@ -7,15 +7,4 @@ released tagged minor version. All previous versions are unsupported.
...
-
-1. Navigate to [Discourse](https://discuss.dockstore.org/) and login.
-2. Select your profile icon, located in the top right corner of the screen.
-3. Select the `mail` icon, located in the dropdown.
-4. Send a message to the `dockstore_admins` group.
-
-Note
-
-> If you are unable to see a New Message button on the mail page, you may be considered a new user and have insufficient privileges. Entering 5 topics and viewing 30 posts over a minimum of 10 minutes will raise your privileges. You will be notified of any privilege changes to your account via the mailbox.
+Use the "Report a vulnerability" button to the upper right to report security vulnerabilities. A helpdesk ticket on [discuss.dockstore.org](https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506) may be an option for privacy complaints or other urgent non-security matters. Both options will be addressed by Dockstore administrators.
```
Comment (kathy-t) [inline SECURITY.md]: I'm unfamiliar with GitHub's reporting mechanism so took me a bit before I realized that the "Report a vulnerability" button is in the Security tab. Maybe we could mention that? Or link to GitHub's docs https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability#privately-reporting-a-security-vulnerability

Comment (kathy-t) [inline SECURITY.md]: Are we expecting users to report vulnerabilities to the specific repo it affects or should they all go to dockstore/dockstore like GitHub issues? 

For notifications, how will we know when there's a new vulnerability reported?

**Review by kathy-t — APPROVED** [review]

File: SECURITY.md (hunk 2 above)
Comment (denis-yuen) [reply SECURITY.md]: Didn't realize that this page could be viewed embedded as a tab parallel to the readme at https://github.com/dockstore/dockstore in addition to the security page, added!

Comment (denis-yuen) [reply SECURITY.md]: > Are we expecting users to report vulnerabilities to the specific repo it affects or should they all go to dockstore/dockstore like GitHub issues? 

I think all repos have the ability to receive reports right now. 
We could limit things, but I think that's ok for now. The UI and/or discourse refer to the main repo. 

> For notifications, how will we know when there's a new vulnerability reported? 

I think it's the normal notification mechanism like with PRs

Comment (denis-yuen) [reply SECURITY.md]: Yup, regular notifications https://github.com/dockstore/dockstore/security/advisories/GHSA-fp43-4wjf-48r4

**Review by hyunnaye — APPROVED** [review]

**Comment by denis-yuen** [comment]
Some PR feedback now that the mechanism has been demoed essentially

**Review by coverbeck — APPROVED** [review]

**Comment by david4096** [comment]
Great this method is much preferred

**Review by david4096 — APPROVED** [review]

**Pull request merged**